paper/src/main/java/vg/civcraft/mc/namelayer/command/commands/ListInvites.java

```diff
@@ -0,0 +1,83 @@
+package vg.civcraft.mc.namelayer.command.commands;
+
+import co.aikar.commands.annotation.CommandAlias;
+import co.aikar.commands.annotation.CommandCompletion;
+import co.aikar.commands.annotation.Description;
+import co.aikar.commands.annotation.Optional;
+import co.aikar.commands.annotation.Syntax;
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.UUID;
+import org.bukkit.ChatColor;
+import org.bukkit.entity.Player;
+import vg.civcraft.mc.namelayer.GroupManager;
+import vg.civcraft.mc.namelayer.GroupManager.PlayerType;
+import vg.civcraft.mc.namelayer.NameAPI;
+import vg.civcraft.mc.namelayer.command.BaseCommandMiddle;
+import vg.civcraft.mc.namelayer.group.Group;
+import vg.civcraft.mc.namelayer.permission.PermissionType;
+
+public class ListInvites extends BaseCommandMiddle {
+
+	@CommandAlias("nllim|listinvites|listinvitedmembers")
+	@Syntax("<group> [rank (e.g: MEMBERS)]")
+	@Description("List the invitees of a group")
+	@CommandCompletion("@NL_Groups @NL_Ranks")
+	public void execute(Player sender, String groupName, @Optional String playerType, @Optional String playerName) {
+		UUID uuid = NameAPI.getUUID(sender.getName());
+
+		Group group = GroupManager.getGroup(groupName);
+		if (groupIsNull(sender, groupName, group)) {
+			return;
+		}
+
+		if (!sender.hasPermission("namelayer.admin")) {
+			if (!group.isMember(uuid)) {
+				sender.sendMessage(ChatColor.RED + "You're not on this group.");
```

Comment:
This is fine to use, however we do want to move to use Kyoris adventure components

Reply:
I assume that's for future compatibility with 1.19+?